src/components/ReactSearchAutocomplete.js

```diff
@@ -40,15 +40,18 @@ export default function ReactSearchAutocomplete(props) {
 
   const [searchString, setSearchString] = useState(inputSearchString)
   const [results, setResults] = useState()
+  const [keyLocation, setKeyLocation] = useState(null)
 
   const callOnSearch = (keyword) => {
     let newResults = []
     if (keyword?.length > 0) {
       newResults = fuseResults(keyword)
       setResults(newResults)
       onSearch(keyword, newResults)
+      setKeyLocation(null)
     } else {
       setResults(newResults)
+      setKeyLocation(null)
```

Comment:
@harveytoro Why not add `setKeyLocation(null)` outside of the if statement since it's called anyway with the same parameters?